artifactory/commands/buildinfo/publish.go

```diff
@@ -91,12 +95,50 @@ func (bpc *BuildPublishCommand) Run() error {
 	if err != nil {
 		return err
 	}
+
+	buildLink, err := bpc.logBuildInfoUiUrl(servicesManager, buildInfo.Started)
+	if err != nil {
+		return err
+	}
+	log.Info("Build info successfully deployed. Browse it in Artifactory under " + buildLink)
+
 	if !bpc.config.DryRun {
 		return utils.RemoveBuildDir(bpc.buildConfiguration.BuildName, bpc.buildConfiguration.BuildNumber, bpc.buildConfiguration.Project)
 	}
 	return nil
 }
 
+func (bpc *BuildPublishCommand) logBuildInfoUiUrl(servicesManager artifactory.ArtifactoryServicesManager, buildInfoStarted string) (string, error) {
+	buildTime, err := time.Parse(buildinfo.TimeFormat, buildInfoStarted)
+	if errorutils.CheckError(err) != nil {
+		return "", err
+	}
+	artVersion, err := servicesManager.GetVersion()
+	if err != nil {
+		return "", err
+	}
+	majorVersion, err := strconv.Atoi(artVersion[0:1])
```

Comment:
Instead of picking up the first character of the version, let's pick the left number.
For example, for version 10.5.10, let's pick up 10, instead of 1. You can split the string by "." to achieve this.